### PR TITLE
Normalize naming of HBH/BBH/4xi8BH to HHS_BH/BBS_BH/4xi8II_BH respectively

### PR DESCRIPTION
**Pull request by babakpst**
Updating HPA types kernel/library logic names: 
1. The name of the following kernels/lib logics changes (HBH->HHS_BH, BBH->BBS_BH. 4xi8BH->4xi8II_BH). 
2. The value_type in the signature of kernels reflects the correct datatype.
3. BBB+HPA automatically converts to BBS_BH in Tensile, the same as HHH+HPA conversion to HHS_BH.

### CODE REVIEW

**Comment by zaliu** [comment]
@babakpst When building rocBLAS, I'm seeing tons of:
`Tensile::WARNING: Inconsistent DataTypes: DataType == f16, DestType == f16, ComputeDataType == f16, but HPA == True (HHH+HPA, no such a type); Converting HHH+HPA to HHS_BH by setting compute data type to f32.
`

**Comment by zaliu** [comment]
@babakpst your code has a couple of flakes:
```
root@5763dc1ee187:/var/tmp/repos/Tensile/Tensile# flake8 .
./KernelWriterConversion.py:25:1: F401 '.DataType.DataType' imported but unused
./KernelWriterAssembly.py:28:1: F401 '.DataType.DataType' imported but unused
```

**Comment by babakpst** [comment]
Thanks, @zaliu. Flakes removed.

File: Tensile/KernelWriterAssembly.py [hunk 1]
```diff
@@ -1334,23 +1333,16 @@ def initKernel(self, kernel, tPA, tPB ):
 
     # already covers: dgemm, cgemm, zgemm, sgemm
     #               : hgemm  + !HPA ([H/H/H] compute = internal = f16)
-    #               : hgemm  +  HPA ([H/H/S] compute = internal = f32) -> new
-    #               : bfgemm +  HPA (compute = internal = f32)
+    #               : hgemm  +  HPA ([H/H/S] or [H/S/S] compute = internal = f32)
+    #               : bfgemm +  HPA ([B/B/S] or [H/S/S] compute = internal = f32)
```
Comment (babakpst) [inline Tensile/KernelWriterAssembly.py:1337]: @nakajee Thanks. This should be B/S/S.

**Review by AlexBrownAMD — APPROVED** [review]
LGTM

**Comment by babakpst** [comment]
Thanks, @zaliu. The failures were due to some tests (mostly sgemm), that are not passing the alpha/beta datatype to ContractionSolution::solve (in ContractionSolution.cpp). The condition that we already have in the function, fixes the issue with the tests! For now, I keep the condition, but I think we need to modify the tests so that we can later remove this condition. I added a TODO comment there.

**Review by zaliu — APPROVED** [review]
A full rocBLAS build and test with this PR on gfx90a passed.

**Comment by zaliu** [comment]
@babakpst Please squash-merge after extended math-CI passes.  Thanks.

**Pull request merged**